lib/lambda_ethereum_consensus/validator/build_block_request.ex

```diff
@@ -14,9 +14,7 @@ defmodule LambdaEthereumConsensus.Validator.BlockRequest do
     attester_slashings: [],
     attestations: [],
     voluntary_exits: [],
-    bls_to_execution_changes: [],
-    eth1_data: nil,
-    execution_payload: nil
```

Comment:
I'm not sure about this change. It might become a parameter explosion in the future.

Reply:
Im not convinced about this argument, I would like to separate the arguments passed from the caller (which are customizable) and the ones generated internally.

Reply:
I'm not thinking of them as "internal", that's why (related to https://github.com/lambdaclass/lambda_ethereum_consensus/pull/967#discussion_r1559865377). Let's merge this PR and discuss this later if/when it becomes a problem.